tests/test_develop.py

```diff
@@ -99,3 +102,22 @@ def test_uninstall(site_packages: Path, conda_pth: Path):
         _uninstall(site_packages, path)
 
         assert list(filter(None, conda_pth.read_text().split("\n"))) == develop_paths
+
+
+def test_get_setup_py(conda_pth: Path):
+    with TemporaryDirectory() as tmpdir:
+        conda_pth = Path(tmpdir)
+        setup_py_path = join(conda_pth, "setup.py")
+        open(setup_py_path, "x").close()
+        result = get_setup_py(str(conda_pth))
+        assert "setup.py" in result
```

Comment:
Use pytest's `tmp_path` instead:

```suggestion
def test_get_setup_py(tmp_path: Path):
    setup_py_path = tmp_path / "setup.py"
    setup_py_path.touch()
    result = get_setup_py(str(tmp_path))
    assert "setup.py" in result
```